extensions/wikia/WallNotifications/WallNotificationControllerBase.class.php

```diff
@@ -276,6 +247,28 @@ protected function fixNotificationURL( $url ) {
 	}
 
 	/**
+	 * @param string $title
+	 * @return string
+	 */
+	protected function shortenTitle( string $title ): string {
+		return $this->getWallHelper()->shortenText( $title, static::NOTIFICATION_TITLE_LIMIT );
```

Comment:
`shortenText` method can be made static in `WallHelper` class, saving object creation and reducing the number of methods (`getWallHelper`). We can also get rid of `$app->wg->Lang` there.